tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "Node",
+    "declaration": true,
+    "module": "CommonJS",
+    "sourceMap": true,
+    "strict": true,
+    "target": "es5",
+    "esModuleInterop": true,
+    "typeRoots": [
+      "./node_modules/@types",
+    ],
+    "outDir": "dist"
+  },
+  "include": [
+    "*.ts"
+  ],
+  "exclude": [
+    "node_modules"
+  ]
```

Comment:
Does `dist` need to be included here? If it's already working, there's no need to change it, I'm just not sure!

Reply:
I did not include `dist` in `exclude `because I did not see them in the other repos I was referring to. But a further check on the official typescript document seems that it suggests adding the value of `outDir` to exclude, in our case `dist `, so I am adding `dist` to `exclude`. All the tests are passing on both scenarios.

Reply:
Cool! Explicit is generally better when it comes to configs